### PR TITLE
Update requirements, cap Sphinx

### PR DESCRIPTION
**Pull request by wsanchez**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -94,7 +95,7 @@ setenv =
     BLACK_LINT_ARGS=--check
 
 commands =
-    black {env:BLACK_LINT_ARGS:} setup.py src
+    black {env:BLACK_LINT_ARGS:} {posargs:setup.py src}
```
Comment (wsanchez) [inline tox.ini:98]: This is to allow one to specify specific files.

File: setup.py [hunk 2]
```diff
@@ -27,11 +27,11 @@
     url=__url__,
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data=dict(hyperlink=["py.typed",],),
+    package_data=dict(hyperlink=["py.typed"]),
     zip_safe=False,
     license=__license__,
     platforms="any",
-    install_requires=["idna>=2.5", 'typing ; python_version<"3.5"',],
+    install_requires=["idna>=2.5", 'typing ; python_version<"3.5"'],
```
Comment (wsanchez) [inline setup.py:34]: Because we're adding `setup.py` to the files Flake8 looks at.

Comment (mahmoud) [reply setup.py:34]: Appreciate the explanation :)

File: tox.ini (hunk 1 above)
Comment (mahmoud) [reply tox.ini:98]: makes sense

**Pull request merged**